test/integration/xfcc_integration_test.cc

```diff
@@ -0,0 +1,243 @@
+#include "xfcc_integration_test.h"
```

Comment:
I don't have an opinion on this. I agree there is a lot of boilerplate but at the same time it's not *that* bad. Whatever you both want is fine with me.